src/background/telemetry/telemetry-event-handler.ts

```diff
@@ -28,6 +28,7 @@ export class TelemetryEventHandler {
 
         const flattenTelemetryInfo: DictionaryStringTo<string> =
             this.flattenTelemetryInfo(telemetryInfo);
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
         this.telemetryClient.trackEvent(eventName, flattenTelemetryInfo);
```

Comment:
Eventually, `publishTelemetry` should become async so this call can be awaited. However, `publishTelemetry` is called by almost every ActionMessageCreator, as well as some actions and store update listeners, so we'll need to make significant changes to the flux infrastructure code first. For now, we'll just disable the rule here, so it's more easily searchable than marking the promise `void`.

Reply:
This makes sense! Can you add the context from this PR comment as a comment in the code next to this line?